packages/ckeditor5-image/src/imageresize/resizeimagecommand.ts

```diff
@@ -77,4 +79,59 @@ export default class ResizeImageCommand extends Command {
 			} );
 		}
 	}
+
+	/**
+	 * Returns image width in specified units.
+	 *
+	 * 	* If image is not selected or command is disabled then `null` will be returned.
+	 * 	* If image is not fully loaded (and it is impossible to determine its natural size) then `null` will be returned.
+	 *  * If `targetUnit` percentage is passed then it will return width percentage of image related to its accessors.
+	 *
+	 * @param targetUnit Unit in which dimension will be returned.
+	 * @returns Parsed dimension with unit.
+	 */
+	public getSelectedImageWidthInUnits( targetUnit: string ): DimensionWithUnit | null {
+		const { editor, isEnabled, value } = this;
+		const { editing } = editor;
+		const imageUtils = editor.plugins.get( 'ImageUtils' );
+
+		if ( !isEnabled || !value ) {
+			return null;
+		}
+
+		const parsedWidth = tryParseDimensionWithUnit( value.width );
+
+		if ( !parsedWidth ) {
+			return null;
+		}
+
+		if ( parsedWidth.unit === targetUnit ) {
+			return parsedWidth;
+		}
+
+		const imageModelElement = imageUtils.getClosestSelectedImageElement( editor.model.document.selection )!;
+		const imageViewElement = editing.mapper.toViewElement( imageModelElement );
+		const imageDOMElement = editing.view.domConverter.mapViewToDom( imageViewElement! );
+
+		if ( !imageDOMElement ) {
```

Comment:
When would this happen? The method returns early if the command has no `value`.

Reply:
Condition removed